VirtoCommerce.Storefront/appsettings.Development.json

```diff
@@ -8,7 +8,11 @@
     }
   },
   "VirtoCommerce": {
-    "Endpoint": {    
+    "CacheEnabled": "false",
```

Comment:
Probably we do not need it in Dev config, as such settings are specific to the user (e.g. Url and SecretKey). User Secrets is the place when we need to specify local settings.
![image](https://user-images.githubusercontent.com/15610352/56948952-23de4800-6b4b-11e9-8ecc-4e296bbbc9a7.png)
So I think it's better to revert this file.

Reply:
ok